data_structures/dict_text_analysis.py

```diff
@@ -0,0 +1,23 @@
+# dictionaries for text analysis in a file.
+
+
+def text_analysis(file_name):
+
+    file1 = open(file_name, "r")
+    d = dict()
+    for line in file1:
+        words = line.split()
+        for word in words:
+            if word in d:
+                d[word] += 1
+            else:
+                d[word] = 1
+    return d
+
+
+if __name__ == "__main__":
+    # file_name = "/Users/rct/src/interview-prep/data_structures/test2.txt"
```

Comment:
Remove commented out code.

Reply:
You have to use a argparser and then set default arguments so that you can override when needed but the script has a default behavior otherwise. Commenting uncommenting code is not the way to do it.

Reply:
Look into argparse if you don't know what it is.